tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py34,py35,py36,pypy,pypy3
 
 [testenv]
 commands =
-    python setup.py -q test -q
-deps = 
-    zope.testing
-    six
+    zope-testrunner --test-path=src
+deps =
+    .[test]
```

Comment:
:+1: for supporting `tox -e coverage` (in a separate PR)

:-1: for including `coverage` in the default `envlist`.